deepchecks/vision/checks/distribution/heatmap_comparison.py

```diff
@@ -0,0 +1,255 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module contains Train Test label Drift check."""
+from typing import Tuple, List, Any, Iterable, Optional
+
+import cv2
+import torch
+from plotly.subplots import make_subplots
+
+from deepchecks.vision.utils.image_functions import numpy_grayscale_to_heatmap_figure, apply_heatmap_image_properties
+
+from deepchecks.core import DatasetKind, CheckResult
+from deepchecks.core.errors import DeepchecksNotSupportedError, DeepchecksValueError
+from deepchecks.vision.base import Context, TrainTestCheck
+from deepchecks.vision.dataset import TaskType
+import numpy as np
+import plotly.graph_objs as go
+
+__all__ = ['HeatmapComparison']
+
+
+class HeatmapComparison(TrainTestCheck):
+    """
+    Check if the average image brightness (or bbox location if applicable) is similar between train and test set.
+
+    The check computes the average grayscale image per dataset (train and test) and compares the resulting images.
+    This comparison may serve to visualize differences in the statistics of the datasets. Additionally, in case of an
+    object detection task, the check will compare the average locations of the bounding boxes between the datasets.
+
+    Parameters
+    ----------
+    classes_to_display : Optional[List[float]], default: None
```

Comment:
Is the class_id or class name? Why not using the `label_id_to_name` from the dataset?